examples/chef/linux/include/CHIPProjectAppConfig.h

```diff
@@ -32,3 +32,11 @@
 
 // Allows app options (ports) to be configured on launch of app
 #define CHIP_DEVICE_ENABLE_PORT_PARAMS 1
+
+// Enable subscriptions synchronization
+#define CHIP_CONFIG_SYNCHRONOUS_REPORTS_ENABLED 1
+
+// ICD configurations
+#define CHIP_CONFIG_ICD_IDLE_MODE_DURATION_SEC 30
```

Comment:
This is a very short interval. This should be configurable on the command line to allow flexibility.

Reply:
The method `ICDConfigurationData::SetModeDurations` allows to do that. It should be overridable from command line parameters beyond the starting default of this config.

Reply:
CHIP_CONFIG_ICD_IDLE_MODE_DURATION_SEC is better to be handled with runtime configuration. thanks

Reply:
Done. PTAL.